### DISCUSSION TITLE
Show averages

### DISCUSSION THREAD

**Discussion by julhac** [iSoron/uhabits#806] [Feature Requests]
I would like to see averages and compare them to my goal. E.g. I have a goal of three times per week: then a feature with a bar indicating the current average and a reference line at three would be helpful.

**Comment by hiqua** [comment]
I think you're supposed to use the score for this kind of stuff. Why is the score insufficient for your use-case?

Of course it would be possible to add this graph after all the others, but there are quite a few already.

See also: #775

**Reply by julhac** [reply]
I see your point and I like the score and how it's being computed. However, I think for a quick idea of how well one has performed since beginning the habit, a normal average would be a more transparent as less sophisticated figure.

**Comment by MarkJeronimus** [comment]
Not sure if this is duplicated as https://github.com/iSoron/uhabits/discussions/1299 but the other exactly sounds like my issue and this one is ambiguous.
